### PR TITLE
Supported components

### PR DESCRIPTION
**Pull request by kellymore**

### CODE REVIEW

File: pages/concepts/drop-in-components.mdx [hunk 1]
```diff
@@ -46,21 +52,189 @@ Dwolla’s Drop-in UI Components are customizable to match the look and feel of
...
+<Image
+  src={createCustomerDropIn}
+  alt="Create a Customer Drop-in Component"
+  width="75%"
+  height="75%"
+/>
+
+### Upgrade an Unverified Customer
+
+The `dwolla-customer-update` component is used to upgrade an Unverified Customer into a Personal Verified Customer. Upgrading an Unverified Customer to a Verified Customer will give them the ability to increase their transaction limits. For more information on the difference between an Unverified and Verified Customer, visit our docs on concepts.
```
Comment (spencerhunter) [inline pages/concepts/drop-in-components.mdx]: @misskellymore , could we add a link here for [visit our docs](/concepts/customer-types/) ?

File: pages/concepts/drop-in-components.mdx [hunk 2]
```diff
@@ -46,21 +52,189 @@ Dwolla’s Drop-in UI Components are customizable to match the look and feel of
...
+<Image
+  src={customerUpgradeDropIn}
+  alt="Upgrade an Unverified Customer Drop-in Component"
+  width="75%"
+  height="75%"
+/>
+
+### Create a Personal Verified Customer
+
+The `dwolla-personal-vcr` component creates Personal Verified Customers on initial onboard. Personal Verified Customers can both send and receive money, as well as hold a balance. A customer can be onboarded with these capabilities as a Personal Verified Customer without needing to start out as an Unverified Customer who can then upgrade. To learn more about the different customer types, visit our docs on concepts.
```
Comment (spencerhunter) [inline pages/concepts/drop-in-components.mdx]: Same as above. Could we link them to the concepts article which covers more on customer types?

File: pages/concepts/drop-in-components.mdx [hunk 3]
```diff
@@ -46,21 +52,189 @@ Dwolla’s Drop-in UI Components are customizable to match the look and feel of
...
+<Image
+  src={createPVCDropIn}
+  alt="Create a Personal Verified Customer Drop-in Component"
+  width="75%"
+  height="75%"
+/>
+
+### Document upload
+
+The `dwolla-document-upload` component can be used after creating a Personal Verified Customer or upgrading an Unverified Customer to a Personal Verified Customer, in order to uploaad a government-issued document might be required to verify a customer’s identity.
```
Comment (spencerhunter) [inline pages/concepts/drop-in-components.mdx]: small spelling update on "upload". I believe this component should work for Business Verified Customers and Personal Verified Customers that are in a document status.

**Review by spencerhunter — CHANGES_REQUESTED** [review]

File: pages/concepts/drop-in-components.mdx (hunk 1 above)
Comment (kellymore) [reply pages/concepts/drop-in-components.mdx]: yes

**Pull request merged**